src/main/java/com/minecolonies/coremod/colony/buildings/AbstractSchematicProvider.java

```diff
@@ -130,6 +148,8 @@ public CompoundNBT serializeNBT()
 
         compound.putBoolean(TAG_DECONSTRUCTED, isDeconstructed);
 
+        BlockPosUtil.write(compound, TAG_PARENT_SCHEM, parentSchematic);
+        BlockPosUtil.writePosListToNBT(compound, TAG_CHILD_SCHEM, new ArrayList<>(childSchematics));
```

Comment:
Near here, for example, is some code that you didn't change that does `getSchematicName() + buildingLevel`, which seems like it would be doing the wrong thing now if you're including the level in the name.  Why not just keep them in separate properties?

Reply:
these are the positions of the parent/child not the schematic names

Reply:
I know that.  I'm referring to several lines above that in the same function.

Reply:
The point being that there are several places that assume the level number is not appended to the schematic name already, so if you're changing that, you need to change those.

Reply:
no you're confusing buildings with the tileentities now, the schematic data is put into the anchors/tileentities, the buildings have their own hard-coded required name to them, which is different from the tile data and doesnt need to be the same aswell

Reply:
Ok, all right then.  It's hard to tell them apart just from looking at the diff.

Reply:
ye its the same method name, just one is on the building which returns hard-coded hut values like "guardtower" and the other is on the TE which returns the schematics filename(which includes the level)